lib/chef/telemetry.rb

```diff
@@ -0,0 +1,47 @@
+require_relative "log"
+require "time" unless defined?(Time.zone_offset)
+require "chef-licensing"
+require_relative "telemetry/null"
+require_relative "telemetry/http"
+
+class Chef
+  class Telemetry
+
+    @@instance = nil
```

Comment:
Do not introduce `@@` variables. A singleton instance variable will work if you're trying to assign this to the class as a whole.

Reply:
This is not needed in this PR and removed it.